gitlab/client.py

```diff
@@ -66,15 +71,14 @@ def __init__(
         http_password: Optional[str] = None,
         timeout: Optional[float] = None,
         api_version: str = "4",
-        session: Optional[requests.Session] = None,
```

Comment:
/cc @JohnVillalovos technically this is a small breaking change, but it's buried so deep into keyword args I can't imagine it being a problem, as people cannot provide multiple tokens in any case. So these would likely be provided as keyword anyway and the order would not be affected.